<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Количество отрезков</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="linesCount">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Длина отрезков (макс.)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="linesLength">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>50px</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="generateLinesButton">
     <property name="text">
      <string>Пересоздать набор отрезков</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWidget" name="moc_drawArea" native="true">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>550</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
